packages/graph-explorer/src/core/connector.ts

```diff
@@ -8,104 +8,97 @@ import { createGremlinExplorer } from "@/connector/gremlin/gremlinExplorer";
 import { createOpenCypherExplorer } from "@/connector/openCypher/openCypherExplorer";
 import { createSparqlExplorer } from "@/connector/sparql/sparqlExplorer";
 import { mergedConfigurationSelector } from "./StateProvider/configuration";
-import { atom, selector, useRecoilValue } from "recoil";
-import { equalSelector } from "@/utils/recoilState";
 import { ConnectionConfig } from "@shared/types";
 import { logger } from "@/utils";
 import { featureFlagsSelector } from "./featureFlags";
 import { Explorer } from "@/connector/useGEFetchTypes";
 import { emptyExplorer } from "@/connector/emptyExplorer";
+import { atom, useAtomValue } from "jotai";
+import { selectAtom } from "jotai/utils";
 
 /**
  * Active connection where the value will only change when one of the
  * properties we care about are changed.
  */
-const activeConnectionSelector = equalSelector({
-  key: "activeConnection",
-  get: ({ get }) => {
-    const config = get(mergedConfigurationSelector);
-    return config?.connection;
-  },
-  equals: (latest, prior) => {
```

Comment:
To achieve similar behavior as `equalSelector` I had to use [`selectAtom` from Jotai](https://jotai.org/docs/utilities/select). When used by itself it worked at runtime, but not with the tests. Wrapping the `selectAtom` in a derived atom fixed the tests and the runtime still works correctly.